.circleci/config.yml

```diff
@@ -208,6 +208,28 @@ jobs:
               apt-get install libtinfo-dev
               bash -c "$PACKCHECK cabal-v1"
         - *save
+  coveralls-ghc-8.8.3:
+      <<: *env
+      steps:
+        - checkout
+        - *preinstall
+        - *restore
+        - run:
+            environment:
+              BUILD: cabal-v2
+              COVERALLS_OPTIONS: "--repo-token=$REPO_TOKEN --coverage-mode=StrictlyFullLines --exclude-dir=test"
+              CABAL_GLOBAL_INSTALL_OPTIONS: " --project-file=cabal.project.coveralls"
```

Comment:
We have a `CABAL_PROJECT` environment variable in packcheck to specify the project file. Won't that work here?

Reply:
I can use it here, because cabal doesn't work properly I had to essentially do it anyway.